src/utils/addresses.ts

```
@@ -141,17 +141,24 @@ export const getContractAddresses = (
-export const getReserveCoinAddress = (chainId: ChainId): `0x${string}` | null =>
-  getContractAddresses(chainId)?.reserveCoin ?? null;
-
-export const getOracleAddress = (chainId: ChainId): `0x${string}` | null =>
-  getContractAddresses(chainId)?.oracle ?? null;
+// Legacy exports for backward compatibility (defaulting to Sepolia testnet)
+export const DJED_ADDRESS = process.env
+  .NEXT_PUBLIC_DJED_CONTRACT as `0x${string}`;
+export const STABLE_COIN_ADDRESS = getStableCoinAddress(11155111);
+export const RESERVE_COIN_ADDRESS = getReserveCoinAddress(11155111);
+export const ORACLE_ADDRESS = getOracleAddress(11155111);
+export const COLLATERAL_ASSET_ADDRESS = getCollateralAssetAddress(11155111);
```

Biome (2.4.7)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)